### PR TITLE
Fix only search bar

### PR DESCRIPTION
**Pull request by amy-lei**
When making the contact cards, I realized I could hardly see 1 card per row bc the header was so big :( Unfortunately i couldnt get stick for just the search bar to work so this wonky fix is to have it fixed on the screen after X amount of scroll. 
Fortunately with the scroll to top PR, this isn't tooooo bad

image

### CODE REVIEW

File: client/src/components/SearchBar.js [hunk 1]
```diff
@@ -1,32 +1,35 @@
...
+            window.removeEventListener('scroll', handleScroll);
+        };
+    }, []);
 
     const handleOnChange = (e) => {
         // TODO: perhaps break at new tags? actually use query?
         setInputValue(e.target.value);
     }
 
-    const tags = query.map((tag, i) => {
```
Comment (yahengsu) [inline client/src/components/SearchBar.js:14]: does it still filter without tags?

or wait what is this for LOL

Comment (amy-lei) [reply client/src/components/SearchBar.js:14]: This was back when i was ambitious lol

Comment (amy-lei) [reply client/src/components/SearchBar.js:14]: Where doing something like `#tag<space>` had this cool look of an actual tag in the search bar and then you could add more tags :')

Comment (yahengsu) [reply client/src/components/SearchBar.js:14]: ooo ok bet lgtm then!

**Review by yahengsu — APPROVED** [review]

**Pull request merged**